components/feedstock/stores.js

```diff
@@ -41,21 +43,32 @@ const Store = ({ dataset, color }) => {
     setTick(timeout)
   }
 
+  const tooltipContent = dataset.public
+    ? ''
+    : 'This dataset is private. Access requires credentials or a Columbia-LEAP JupyterHub server.'
```

Comment:
@jbusecke, curious to hear your take on this wording?


![Screenshot 2024-05-07 at 15 25 51](https://github.com/carbonplan/leap-data-catalog/assets/13301940/42e3b89d-d579-4a26-b11f-0cfbc4e6995e)

Reply:
Not a huge fan of 'private' Can we just use the second sentence? I am also still in need to make those buckets requester pays, so we could say that?

Reply:
sounds good. i'm going to remove the first sentence